src/tsd.c

```diff
@@ -363,8 +363,11 @@ _malloc_thread_cleanup(void) {
 }
 #endif
```

Comment:
Shall we also include the register() below into this #if (since we are now exporting the symbol)?

Reply:
Yes, I think so, but it has some other stuff that depends on it and needs to be changed. It's the right thing to do, though, so I'll change it and update the PR.